### DISCUSSION TITLE
Negative control wells in smart-seq protocol

### DISCUSSION THREAD

**Discussion by abspangler13** [milaboratory/mixcr#1866] [Q&A]
Hello,

When running smart-seq, we use wells H7 through H12 of a 96-well plate as negative controls (do not contain cells) to help us identify contaminating sequences. Is there any way to incorporate this into the MiXCR workflow such that it recognizes those wells as negative control wells? I'm currently using `mixcr -Xmx100g analyze smart-seq2-vdj` to analyze this data.

**Comment by mizraelson** [comment]
Hi, unfortunately, there isn’t a feature to mark wells as negative controls. You can analyze them along with the other wells and check if any clonotypes are present.
